cmd/restic/integration_test.go

```diff
@@ -69,7 +70,9 @@ func testRunBackup(t testing.TB, dir string, target []string, opts BackupOptions
 		defer cleanup()
 	}
 
-	rtest.OK(t, runBackup(opts, gopts, term, target))
+	failureOccured := runBackup(opts, gopts, term, target) != nil
```

Comment:
Please adjust that test function to pass up the returned error and then check for the expected error type in TestBackupErrors.